src/api/kube.ts

```diff
@@ -1304,19 +1304,29 @@ export class KubeHelper {
 
       return crs[0]
     } catch (e) {
+      if (e.response && e.response.statusCode === 404) {
+        throw this.wrapK8sClientError('Custom Resource "checlusters.org.eclipse" not found.')
```

Comment:
Is it better to `return undefined` ?

Reply:
Apparently is better to return undefined and don't catch all statusCode.

Reply:
Is better to return undefined because if we return [] or something else https://github.com/che-incubator/chectl/blob/master/src/tasks/installers/operator.ts#L340 will not work properly